jwql/tests/test_cosmic_ray_monitor.py

```diff
@@ -0,0 +1,121 @@
+import pytest
```

Comment:
These tests look good at first glance! 

Could you add a module docstring to this file?

Reply:
Added as well as fixed some pep8 stuff